### ISSUE TITLE
Fix rules on lemonde.fr website

### ISSUE THREAD

**Issue by Manu1400** [EFForg/https-everywhere#10962]
1/ Install on Google Chrome HTTPS everywhere extensions
2/ Go to http://www.lemonde.fr/ homepage
3/ The CSS https://asset.lemde.fr/data/0.14.2/css/cadre-page.css ressource can be downloaded with HTTPS.

Note: same issue on website http://libresglisses.blog.lemonde.fr/2017/07/08/surf-une-ceremonie-paddle-out-a-la-memoire-de-jack-oneill/

Please, don't force HTTPS on lemde.fr domain and subdomains.

**Comment by Bisaloo** [comment]
Okay, I take the blame for this one. I enabled the `lemde.fr` ruleset for everyone.

`asset.lemde.fr` wasn't included in the ruleset because it is mismatched.

But `s1.lemde.fr` links to `//asset.lemde.fr/data/0.14.2/css/cadre-page.css` in a tricky way and I overlooked that.

```html
<link rel="stylesheet" type="text/css" href="//s1.lemde.fr/bootstrap/www/ebd57164de17336ace1418284fb6434f.css?m[]=normalize%40screen&m[]=fonts%40screen&m[]=grille%40screen&m[]=base%40screen&m[]=header%40screen&m[]=footer%40screen&m[]=rubrique%40screen&m[]=article%40screen&m[]=friends%2Fpaves%40screen&m[]=col_droite%40screen&m[]=ticker%40screen&m[]=couleurs%40screen&m[]=marketing%2Fmeter%40screen&m[]=evenementiel%2Fcoupe-du-monde%40screen&m[]=elections%2Famericaines%40screen&m[]=friends%2Fnewsweb%2Flivescore%40screen&m[]=friends%2Fnewsweb%2Flivescore_tennis%40screen&m[]=%2F%2Fasset.lemde.fr%2Fdata%2F0.14.2%2Fcss%2Fcadre-page%40screen&m[]=une%40screen&m[]=friends%2Fnewsweb%2Fbase%40screen&m[]=friends%2Fnewsweb%2Fdatabox%40screen">
```
I have submitted a fix in #10964. It should be included in the next release. In the meantime, you can disable `lemde.fr` rules by clicking on the ruleset in the dropdown menu.

**Comment by J0WI** [comment]
closed by #10964
